api/src/main/java/marquez/api/filter/exclusions/ExclusionsConfig.java

```diff
@@ -0,0 +1,15 @@
+package marquez.api.filter.exclusions;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Getter;
+
+public class ExclusionsConfig {
+  @Getter @JsonProperty public NamespaceExclusion namespaces;
+
+  public static class NamespaceExclusion {
+    @Getter @JsonProperty public boolean onRead;
+    @Getter @JsonProperty public boolean onWrite;
+    @Getter @JsonProperty public String patterns;
```

Comment:
Is this a single pattern or a list of patterns separated in some way?

Reply:
@pawel-big-lebowski yes, sorry about that, I forgot to include the proper documentation. I added a section to the FAQ to explain the usage, I'd be happy to move it to another place that is more suitable if needed